packages/fxa-settings/src/components/PasswordStrengthBalloon/en.ftl

```diff
@@ -0,0 +1,7 @@
+## PasswordStrengthBalloon component
+
+password-strength-balloon-heading = Password requirements
+password-strength-balloon-min-length = At least 8 characters
+password-strength-balloon-not-email = Not your email address
+password-strength-balloon-not-common = Not a commonly used password
+password-strength-balloon-stay-safe-tips = Stay safe {"\u2014"} Don’t reuse passwords. See more tips to <LinkExternal>create strong passwords</LinkExternal>.
```

Comment:
```suggestion
password-strength-balloon-stay-safe-tips = Stay safe — Don’t reuse passwords. See more tips to <LinkExternal>create strong passwords</LinkExternal>.
```

Fluent supports unicode characters (including em dashes) so best to use the actual character so localizers can make an informed choice in how to render the sentence in translation.

Reply:
I was following the guidance from https://projectfluent.org/fluent/guide/special.html which had an example about dashes, because it might not be visually obvious if this is an em dash or en dash:

```
A similar approach will make it clear which dash character is used in the following example:

# The dash character is an EM DASH but depending on the font face,
# it might look like an EN DASH.
which-dash1 = It's a dash—or is it?

# Using a Unicode escape sequence makes the intent clear.
which-dash2 = It's a dash{"\u2014"}or is it?
```

I'm fine with switching to a regular character though if the above isn't relevant in this case!

Reply:
In this case I think using the actual punctuation is clearer. Pontoon also allows translators to pull down the punctuation from the source string if necessary ([example](https://pontoon.mozilla.org/ja/firefox-accounts/all-resources/?search=%E2%80%94&string=246196)).